internal/pkg/cli/task_run.go

```diff
@@ -376,13 +379,25 @@ func (o *runTaskOpts) Execute() error {
 		return err
 	}
 
+	if o.env == "" {
+		has, err := o.defaultClusterGetter.HasDefaultCluster()
```

Comment:
You should change has to be more descriptive. At first glance, it's hard to understand what has represents